integration-cli/docker_api_containers_test.go

```diff
@@ -552,7 +552,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	if cmd != "{[/bin/sh -c touch /test]}" {
```

Comment:
Yup - which means, line like: https://github.com/docker/docker/blob/master/graph/service.go#L46 would need to be replaced with code that converts the "internal" runconfig.Config into something that matches what we want people to see (on the wire and via "inspect -f")